forms-flow-components/src/components/CustomComponents/Radio.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import Form from "react-bootstrap/Form";
+
+interface RadioOption {
+    label: string;
+    onClick: () => void;
+  }
+  
+  interface CustomRadioButtonProps {
+    items: RadioOption[];
+    dataTestid?: string;
+    ariaLabel?: string;
+  }
+
+export const CustomRadioButton: React.FC<CustomRadioButtonProps> = ({
+  items,
+  dataTestid = "",
+  ariaLabel = "",
+}) => {
+  return (
+    <Form className="custom-radio-button mb-3" data-testid={dataTestid} aria-label={ariaLabel}>
```

Comment:
Please move this data-testid attribute to an attribute in <Form.Check> tag . 
As it is a list , pls give something like  data-testid={`inline-radio-${index + 1}`}

Reply:
Updated